src/Runner.Listener/JobDispatcher.cs

```diff
@@ -13,6 +13,8 @@
 using GitHub.Runner.Common;
 using GitHub.Runner.Sdk;
 using GitHub.Services.WebApi.Jwt;
+using System.Text.RegularExpressions;
```

Comment:
consider sorting these (i see other system at the top)

Reply:
fixed